lib/category/category_page.dart

```diff
@@ -39,20 +42,27 @@ class CategoryView extends StatelessWidget {
               child: CircularProgressIndicator(),
             );
           case Status.success:
-            return ListView(
-              children: state.categories
-                  .map(
-                    (category) => ListTile(
-                      title: Text(category.name),
-                      onTap: () {
-                        context.router.pushNamed(
+            return SingleChildScrollView(
+              child: Column(
```

Comment:
No need for a `Column` widget, we can use a `Padding` between `SingleChildScrollView` and the `FlexGridLayout`